### PR TITLE
CDPS-298 - update date range labels

### PR DESCRIPTION
**Pull request by jamesgrant-moj**

### CODE REVIEW

File: server/services/alertsPageService.ts [hunk 1]
```diff
@@ -28,15 +28,15 @@ export default class AlertsPageService {
...
-      errors.push({ text: `'Date from' must be today or in the past`, href: '#from' })
+      errors.push({ text: `'Date from (earliest)' must be today or in the past`, href: '#from' })
     }
 
     if (to && !isRealDate(to)) {
-      errors.push({ text: `'Date to' must be a real date`, href: '#to' })
+      errors.push({ text: `'Date to (latest)' must be a real date`, href: '#to' })
     } else if (to && from && isBefore(parseDate(to), parseDate(from))) {
-      errors.push({ text: `'Date to' must be after or the same as 'Date from'`, href: '#to' })
+      errors.push({ text: `'Date to (latest)' must be after or the same as 'Date from'`, href: '#to' })
```
Comment (davidmiddletonmoj) [inline server/services/alertsPageService.ts:39]: Should add ` (earliest)` to `Date from`

File: server/services/caseNotesService.ts [hunk 2]
```diff
@@ -34,9 +34,9 @@ export default class CaseNotesService {
     }
 
     if (endDate && !isRealDate(endDate)) {
-      errors.push({ text: `'Date to' must be a real date`, href: '#endDate' })
+      errors.push({ text: `'Date to (latest)' must be a real date`, href: '#endDate' })
     } else if (endDate && startDate && isBefore(parseDate(endDate), parseDate(startDate))) {
-      errors.push({ text: `'Date to' must be after or the same as 'Date from'`, href: '#endDate' })
+      errors.push({ text: `'Date to (latest)' must be after or the same as 'Date from'`, href: '#endDate' })
```
Comment (davidmiddletonmoj) [inline server/services/caseNotesService.ts:39]: Should add  (earliest) to Date from

**Review by davidmiddletonmoj — COMMENTED** [review]
Just a couple of label changes

**Comment by jamesgrant-moj** [comment]
@davidmiddletonmoj ready for review.

**Review by davidmiddletonmoj — APPROVED** [review]
LGTM

**Pull request merged**